bip-XXXX.mediawiki

```diff
@@ -0,0 +1,140 @@
+<pre>
+  BIP: ???
+  Layer: Applications
+  Title: Signatures of Messages using Bitcoin Private Keys
+  Author: Christopher Gilliard &ltchristopher.gilliard@gmail.com&gt
```

Comment:
A literal < and > should be used  here.

Reply:
Fixed.

Reply:
Also updated to BIP 137 in this section and changed file name. Should I do another pull request?

Reply:
No reason for another PR...